paraatm/HPMM/hpmm.py

```diff
@@ -0,0 +1,101 @@
+#! /usr/bin/python3 python3
+# -*- coding: utf-8 -*-
+
+"""
+@Author: Jiawei Chen
+@Date: 2020-05-20
+This Python script is used for HPMM (Human Performance Monitoring Module)
+The script outputs predicted loss of separation as an indicator of air traffic controller's performance
+The script can also check compliance for each command
+@Last Modified by: Jiawei Chen
+@Last Modified date: 2020-05-26
+"""
+from typing import Any, Union
+
+from jpype import *
+from array import *
+import pandas as pd
+import numpy as np
+import scipy.stats as sts
+import os
+import time
+import sys
+import datetime
+from shutil import copyfile
+from shutil import rmtree
+
+import matplotlib.pyplot as plt
+from pandas import Series, DataFrame
+from pandas.io.parsers import TextFileReader
+from scipy.stats import norm
+from sklearn.preprocessing import StandardScaler
+from sklearn.model_selection import train_test_split
+import catboost as cb
+from catboost import CatBoostRegressor
+from catboost import Pool
```

Comment:
Please remove all imports that are not actually used in this file.  Based on a quick check, it appears that most of these (e.g., `sts`, `os`, `time`, `sys`, `datetime`, etc.) are not used.